Add features and fix

Rushaway opened the pull request:
Add Features as ban origin of bans / comms
Imrpove UI 
Fix some text errors
Fix dashboard

Rushaway committed:
Improved option to edit Bans

Fix https://github.com/aXenDeveloper/sourcebans-web-theme-fluent/issues/31

Rushaway committed:
Fix Display and text error in admin edit comms

Rushaway committed:
GameTracker Suppport

Add GameTracker Banner only in Servers Page and on desktop only.

Rushaway committed:
Added Origin of bans display

Add where the ban was made from.

Rushaway committed:
Fix global UI + text + comms block origin support

Rushaway committed:
Fixed global UI

Rushaway committed:
Fix Comms counter

Comms counter was displaying the wrong total function.

Rushaway committed:
Improve display

Rushaway committed:
up version

Rushaway committed:
Feat and Fix CSS

aXenDeveloper reviewed (changes requested):
Please use prettier plugin for better formatting code.

Rushaway committed:
Removed useless content

Rushaway committed:
Fix mistake

aXenDeveloper commented:
Please remove `center` tag form this PR.

Rushaway committed:
Remove old center balise

Rushaway commented:
> Please remove all `center` tag form this PR.

Done.

Rushaway committed:
Fix Buttons can be see without be logged

+ Disable buttons if not on desktop for responsive bug. (temp fix)
(Have to add a buttons for if user is not logged, if no buttons the display is broken..)

Rushaway commented:
https://github.com/aXenDeveloper/sourcebans-web-theme-fluent/pull/50/commits/05b9b08b6e445310f086821c83d8435cf1cd38c5 is fixing https://github.com/aXenDeveloper/sourcebans-web-theme-fluent/issues/51

crashzk commented:
I believe it can be merged now, right @aXenDeveloper.

@Rushaway I just have a question regarding GameTracker support. If the server is not registered in GameTracker, what happens? Doesn't show anything or shows an error?

Rushaway commented:
@crashzk It will show a banner of GameTracker who said the IP isn't registered in GameTracker.

Caaine commented:
not good :S

crashzk commented:
> @crashzk It will show a banner of GameTracker who said the IP isn't registered in GameTracker.

I understand, I believe that the support for GameTracker is good, my opinion is that it should have it, but give it a way for the user to have control over it in the end, he chooses in the sourcebans options whether to activate it or not, as he has the option to show the admin that banned or not to players.

In any case, it is already a good start to have support for it.

Rushaway commented:
@crashzk Can be done, but need to edit the core file of Sourcebanspp.
It's a theme feature, not a global support from Sourcebanspp. So will not be implanted until sourcebanspp add this feature too as native.
People are free to remove or comments this code part.

crashzk commented:
> @crashzk Can be done, but need to edit the core file of Sourcebanspp. It's a theme feature, not a global support from Sourcebanspp. So will not be implanted until sourcebanspp add this feature too as native. People are free to remove or comments this code part.

I see, they've actually done a PR on sourcebans-pp too: https://github.com/sbpp/sourcebans-pp/pull/718

In any case for me even without having this feature I would still use it.

aXenDeveloper commented on sourcebans-web-theme-fluent/page_servers.tpl:
```diff
@@ -35,6 +35,7 @@
                             <tr id="server_{$server.sid}:desc" class="table_hide">
                                 <td colspan="8">
                                     <div class="collapse_content" id="sinfo_{$server.sid}">
+                                    <p class="responsive_show:desktop" align="center"><a href="https://www.gametracker.com/server_info/{$server.ip}:{$server.port}/"><img style="width: auto; height: auto;" src='https://cache.gametracker.com/server_info/{$server.ip}:{$server.port}/banner_560x95.png'/></a></p>
```
Delete this. Not everyone wants this functionality.

Gametracker is very slow to load which may slow down the page loading speed

Rushaway committed:
removed gametracker support
